_pages/ato.md

```diff
@@ -18,12 +18,22 @@ If you are publishing a new site through [Federalist](../infrastructure/#federal
 
 *Note: Technically, static site builders are just adding a collection of pages in an existing system. Therefore, from an ATO perspective, "sites" created through Federalist or 18F Pages remain within the respective security boundary, and thus ATO.*
 
-### Re-authorization
+### How to get re-authorization (get a new ATO)
 
-If a previous ATO expires or the system changes substantively, it'll need a new ATO. Examples for triggers for a new ATO include changes to:
+If your system already has an ATO, there are a couple cases where you'll need to go through the ATO process again: if you plan to make a substantive change, or if your ATO is going to expire.
+
+#### System changes that require a new ATO
+
+Examples of system changes that may require a new ATO include changes to:
 
 * Encryption methodologies
 * Administrative functionality within the application
 * The kinds of information you store (e.g. [personally identifiable information](../security/pii/))
 
-The 18F Infrastructure team will make this determination – please [open a new issue](https://github.com/18F/Infrastructure/issues/new?title=ATO+re-authorization+for+%3Cproject%3E?) if you think you have made a change that may require a new ATO. Even if you're not sure, doesn't hurt to ask!
+The 18F Infrastructure team determines whether a system needs a new ATO. If you're planning a change that you think may require a new ATO, please [open a new issue in the Infrastructure repository](https://github.com/18F/Infrastructure/issues/new?title=ATO+re-authorization+for+%3Cproject%3E?) to explain your planned change, so they can evaluate whether it needs a new ATO.
+
+If you'll need a new ATO, follow the usual steps for getting an ATO, starting with the [the checklist](checklist/). You may be able to reuse and update your existing ATO materials.
+
+#### Replacing an expiring ATO with a new one
+
+If your current ATO is going to expire, you'll need to replace that ATO with a new one. Follow the usual steps for getting an ATO, starting with the [the checklist](checklist/). You may be able to reuse and update your existing ATO materials.
```

Comment:
I went into this and reworked it a bit, but I still ended up with the repeated information. I think it's fine! My imagined user is visually skipping directly to the section that looks relevant to her, so a bit of repeated info doesn't hurt (so that she doesn't miss it).